lib/pages/action/ActionInfo.dart

```diff
@@ -110,6 +110,7 @@ class _ActionInfoState extends State<ActionInfo> with WidgetsBindingObserver {
                                   GestureDetector(
                                       onTap: () {
                                         // TODO This must go to the causes info page
+                                        // For now this can be the home explore page filtered by a single cause
```

Comment:
Really?

Reply:
Oh no this is legit. I will create a ticket for this. https://tree.taiga.io/project/jelgar-now-u-1/us/166?kanban-include_attachments=1&kanban-include_tasks=1&kanban-epic=174902&kanban-status=2500908